app/src/main/java/me/saket/dank/ui/accountmanager/AccountManagerRepository.java

```diff
@@ -0,0 +1,51 @@
+package me.saket.dank.ui.accountmanager;
+
+import android.annotation.TargetApi;
+import android.app.Application;
+import android.content.pm.ShortcutManager;
+import android.database.sqlite.SQLiteDatabase;
+import android.os.Build;
+
+import androidx.annotation.CheckResult;
+
+import com.squareup.sqlbrite2.BriteDatabase;
+import dagger.Lazy;
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+
+import java.util.List;
+import javax.inject.Inject;
+
+@TargetApi(Build.VERSION_CODES.N_MR1)
+public class AccountManagerRepository {
+
+  private final Application appContext;
```

Comment:
Unused property. I suggest following suggestions in Android Studio (should be enabled by default) which catch issues similar to this one.